### PR TITLE
Add/external integrations icons

### PR DESCRIPTION
**Pull request by j-mnizhek**
## Motivation and Context

Added the following list of colored icons according to the request we have:
`Ahrefs`, `Microsoft`, `Moz`, `Shapchat`, `Amazon`, `CallRailColored`, `MailChimp`, `MatomoColored`, `LinkedInColored`, `TikTokColored`, `TikTokColoredInvert`, `Salesforce`.

## How has this been tested?

Manually.

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Nice improve.

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have updated the documentation accordingly or it's not required.
- [ ] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [ ] I have added new unit tests on added of fixed functionality.

### CODE REVIEW

**Review by ilyabrower — APPROVED** [review]

File: website/docs/style/icon/icons-list.js [hunk 1]
```diff
@@ -4305,6 +4340,60 @@ const iconsList = {
...
+      size: ['l', 'm'],
+      group: 'Color',
+      tags: [
+        'microsoft',
+        'ms',
+        'micro',
+        'soft',
+        'office',
+        'микрософт',
+        'мелкософт',
```
Comment (sheila-semrush) [inline website/docs/style/icon/icons-list.js:4391]: is it appropriate? :)
```suggestion
        'мелкософт',
```

File: website/docs/style/icon/icons-list.js [hunk 2]
```diff
@@ -4305,6 +4340,60 @@ const iconsList = {
...
+        'логотип',
+        'лого',
+        'matomo',
+        'матомо',
+        'веб',
+        'web',
+        'analytics',
+        'аналитика',
+        'software',
+        'софт',
```
Comment (sheila-semrush) [inline website/docs/style/icon/icons-list.js:4377]: a bit surprised to see the 'software' and 'софт' keywords here, is it intentional?

**Review by sheila-semrush — APPROVED** [review]
Everything looks great!
Left two small comments.

File: website/docs/style/icon/icons-list.js (hunk 1 above)
Comment (j-mnizhek) [reply website/docs/style/icon/icons-list.js:4391]: I'll remove these old guys 😅

File: website/docs/style/icon/icons-list.js (hunk 2 above)
Comment (j-mnizhek) [reply website/docs/style/icon/icons-list.js:4377]: Intentionally, yep 😎

**Pull request merged**